R/R/prophet.R

```diff
@@ -443,9 +447,14 @@ set_changepoints <- function(m) {
       }
     }
   } else {
-    # Place potential changepoints evenly through the first 80 pcnt of
+    # Place potential changepoints evenly through the first changepoint.range proportion of
     # the history.
-    hist.size <- floor(nrow(m$history) * .8)
+    if (m$changepoint.range > 1 || m$changepoint.range <= 0){
+      m$changepoint.range <- .8
+      message('changepoint.range greater than 1 or less than equal to 0. Using ',m$changepoint.range)
+    }
```

Comment:
This block should be replaced by a check inside `validate_inputs` that raises an error if not valid, like in the Python version in this commit.